async-query-core/src/test/java/org/opensearch/sql/spark/utils/SQLQueryUtilsTest.java

```diff
@@ -404,15 +407,40 @@ void testAutoRefresh() {
 
   @Test
   void testValidateSparkSqlQuery_ValidQuery() {
-    String validQuery = "SELECT * FROM users WHERE age > 18";
-    List<String> errors = SQLQueryUtils.validateSparkSqlQuery(validQuery);
+    DataSource dataSource = Mockito.mock(DataSource.class);
```

Comment:
Can you use `@Mock` annotation instead?

Reply:
isn't @Mock only used for class level variables?
If yes, is it necessary to declare this variable at class level? I am using `dataSource` variable only in 2-3 methods.

Reply:
You can define dataSource with `@Mock` annotation once, and can use it in multiple test cases. MockitoExtension will initialize for each tests.

Reply:
done